gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/InMemoryGlobberFileSystem.java

```diff
@@ -84,19 +84,25 @@ private InMemoryGlobberFileSystem(Path workingDirectory, Collection<FileStatus>
     }
   }
 
-  /** @inheritDoc */
+  /**
```

Comment:
What got changed in this file? If it is only formatting, I think it can be moved out of this PR.

Reply:
This is formatting change after update of `google-java-fromat` - without it build fails with this update.